### PR TITLE
RELATED: FET-167 Do not proxy if host not set

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: lib/middleware/request_proxy.js [hunk 1]
```diff
@@ -7,6 +7,11 @@ module.exports = function(host) {
     var proxy = httpProxy.createProxyServer({});
 
     var requestProxy = function(req, res) {
+        if (!currentHost) {
+            res.sendStatus(404);
+            return;
```
Comment (petrjanu) [inline lib/middleware/request_proxy.js]: What about to add body here with some text like  "Resource not found as proxy was started without host option"?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: lib/middleware/request_proxy.js (hunk 1 above)
Comment (pbenes) [reply lib/middleware/request_proxy.js]: done

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/grunt-grizzly-merge-compliance-check/2/ : SUCCESS in 22s

**Pull request merged**